lib/excoveralls/stats.ex

```diff
@@ -226,18 +226,44 @@ defmodule ExCoveralls.Stats do
   Exit the process with a status of 1 if coverage is below the minimum.
   """
   def ensure_minimum_coverage(stats) do
-    coverage_options = ExCoveralls.Settings.get_coverage_options
+    coverage_options = ExCoveralls.Settings.get_coverage_options()
     minimum_coverage = coverage_options["minimum_coverage"] || 0
-    if minimum_coverage > 0, do: check_coverage_threshold(stats, minimum_coverage)
+
+    check_coverage_threshold(stats, minimum_coverage)
+  end
+
+  defp check_coverage_threshold(stats, minimum_coverages) when is_map(minimum_coverages) do
```

Comment:
Could you help me adding some unit tests which covers this nested definition scenario? (it helps maintaining the code and ensuring that will not break).

Reply:
Of course!

Reply:
I've just added some unit tests for the minimum file coverage function! Please let me know what you think :heart:

Reply:
Thanks!